SortApp/SortApp.BL/DataAdapters/CreatorDataAdapterData.cs

```diff
@@ -36,7 +36,8 @@ public SqlDataAdapter CreateAdapter(SqlConnection connection)
 			//
 			// InsertCommand.
 			//
-			SqlCommand cmdInsert = new SqlCommand("INSERT INTO Data(OriginalData, SortedData, Name) VALUES(@OriginalData, @SortedData, @Name)", connection);
+			SqlCommand cmdInsert = new SqlCommand("INSERT INTO Data(OriginalData, SortedData, Name, KindOfSortingAlgorithm) " +
```

Comment:
I think we could use verbatim strings instead of concatenation.
Applies to lines below.